src/components/AgreementSections/Info.js

```diff
@@ -77,15 +76,11 @@ export default class Info extends React.Component {
             </KeyValue>
           </Col>
         </Row>
-        <Row>
-          <Col xs={12}>
-            <KeyValue label={<FormattedMessage id="ui-agreements.agreements.currentPeriodNote" />}>
-              <div data-test-agreement-current-period-note style={{ whiteSpace: 'pre-wrap' }}>
-                {currentPeriodNote || '-'}
-              </div>
-            </KeyValue>
-          </Col>
-        </Row>
+        <KeyValue label={<FormattedMessage id="ui-agreements.agreements.currentPeriodNote" />}>
+          <div data-test-agreement-current-period-note style={{ whiteSpace: 'pre-wrap' }}>
+            {agreement?.currentPeriod?.note ?? '-'}
```

Comment:
I actually completely forgot about this, but reviewing another PR reminded me: we now have a [`NoValue` component in stripes-components](https://github.com/folio-org/stripes-components/blob/d6211488ebde5bf1f18d2dd04758cf580310e6ac/lib/NoValue/NoValue.js#L7).

The nice thing about that component is that it sets the `aria-label` so that a screenreader speaks out "no value set" rather than "dash".

Reply:
So you could do something like `?? <NoValue />`

Reply:
even cooler
i made the changes for all values, hope that's ok